Libraries/LibGUI/FileSystemModel.cpp

```diff
@@ -425,6 +430,17 @@ Icon FileSystemModel::icon_for_file(const mode_t mode, const String& name) const
         return m_filetype_html_icon;
     if (name.to_lowercase().ends_with(".png"))
         return m_filetype_image_icon;
+    if (name.to_lowercase().ends_with(".cpp"))
```

Comment:
```suggestion
    if (name.to_lowercase().ends_with(".cpp") || name.to_lowercase().ends_with(".h"))
```
(Probably ought to extract all the `to_lowercase()` calls to a single one too if this keeps growing.)

Reply:
I planned to have a different icon  for .h